content/posts/2021-08-30-engineering-culture/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Our Engineering Culture
+author: Gustavo Barbosa
+twitter: gustavocsb
+layout: post
+lang: en
+path: /blog/our-engineering-culture
+date: 2021-08-30
+comments: true
+---
+Hello, everyone!
+
+We are happy to announce that today we are making our [engineering culture](https://github.com/loadsmart/culture) **open source**.
+
+There you’ll find several articles describing how we operate as an Engineering organization. Content varies from hiring and onboarding to architecture guidelines and tech stack. This is something that we were already sharing with new members through onboarding sessions and with candidates we would reach out to, but the information, in general, was very scattered.
+
+When we decided to make this move, there were mainly two goals in mind:
+1. Create a single, centralized place for our engineers to look at and remember our main practices (internal alignment)
+2. Show to potential new candidates a bit of our day-to-day work so that, if there’s identification, they might want to join the mothership (external exposure)
```

Comment:
Not sure what is meant by `if there’s identification` in this context

Reply:
That was indeed weird. I've changed it a bit. Thanks @alexantoniomoran